<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpAndAway</class>
 <widget class="QMainWindow" name="UpAndAway">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpAndAway</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Datei</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="JSONErstellenButton">
             <property name="text">
              <string>JSON erstellen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="readFileButton">
             <property name="text">
              <string>Datei einlesen</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>159</width>
          <height>288</height>
         </size>
        </property>
        <property name="title">
         <string>Suche 체ber Buchungs-ID</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Buchungs-ID eingeben:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="idInputSpinBox">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>1234567</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>135</width>
             <height>192</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>135</width>
             <height>192</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Suche 체ber Kunden-ID</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Kunden-ID eingeben:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>391</height>
       </size>
      </property>
      <property name="title">
       <string>Daten 채ndern</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>208</width>
         <height>332</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Id</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Startdatum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Enddatum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Preis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Startflughafen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Zielflughafen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Firma</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Hotel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Stadt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Abholstation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>R체ckgabestation</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item>
           <widget class="QDateEdit" name="fromDateEdit"/>
          </item>
          <item>
           <widget class="QDateEdit" name="toDateEdit"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="priceSpinBox">
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fromDestinationLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="toDestinationLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="companyLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="hotelLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="townLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="pickupStationLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="returnStationLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
